interaction.karma.conf.js

```diff
@@ -0,0 +1,36 @@
+const webpackConfig = require('./webpack.config');
+const path = require('path');
+
+delete webpackConfig.entry;
+
+module.exports = (config) => {
+  config.set({
+    browsers: ['ChromeHeadless'],
+    frameworks: ['mocha', 'sinon-chai'],
+    reporters: ['spec', 'coverage-istanbul'],
+    files: [
+      'test/*.ts',
+      'test/interaction/*.test.ts'
+    ],
+    preprocessors: {
+      'test/*.ts': ['webpack'],
+      'test/interaction/*.test.ts': ['webpack']
```

Comment:
I know this was in the other one, but I'm pretty sure this is redundant. The first one should cover it. Double-check, though.